README.md

```diff
@@ -35,7 +35,7 @@ to run the project locally, clone the project using the command
 navigate in to the project directory, from GitHub,
 to test or run the application double click on the `index.html` file to open in your browser
 
-## Authors
+## Author
 
 👤 **Author**
 
```

Comment:
- [ ] Please, consider fixing the broken screenshot in your README file 👍 
<img width="583" alt="Screen Shot 2022-03-04 at 22 35 29" src="https://user-images.githubusercontent.com/49064106/156822047-c20dfd86-ddb7-49c1-81a7-f9940ddb0161.png">